zcash_client_backend/CHANGELOG.md

```diff
@@ -160,8 +173,38 @@ and this library adheres to Rust's notion of
   - `decode_extended_spending_key`
   - `decode_extended_full_viewing_key`
   - `decode_payment_address`
-- `data_api::wallet::create_spend_to_address` has been modified to use a unified
-  spending key rather than a Sapling extended spending key.
+- `zcash_client_backend::wallet::SpendableNote` is now parameterized by a note
+  identifier type and has an additional `note_id` field that is used to hold the
+  identifier used to refer to the note in the wallet database.
+- `zcash_client_backend::data_api::BlockSource` has been moved to the
+  `zcash_client_backend::data_api::chain` module.
+- The types of the `with_row` callback argument to `BlockSource::with_blocks`
+  and the return type of this method have been modified to return
+  `zcash_client_backend::data_api::chain::error::Error`.
+- `zcash_client_backend::data_api::testing::MockBlockSource` has been moved to
+  `zcash_client_backend::data_api::chain::testing::MockBlockSource` module.
+- `zcash_client_backend::data_api::chain::{validate_chain, scan_cached_blocks}`
+  have altered parameters and result types. The latter have been modified to
+  return`zcash_client_backend::data_api::chain::error::Error` instead of
+  abstract error types. This new error type now wraps the errors of the block
+  source and wallet database to which these methods delegate IO operations
+  directly, which simplifies error handling in cases where callback functions
+  are involved.
+- `zcash_client_backend::data_api::error::ChainInvalid` has been moved to
+  `zcash_client_backend::data_api::chain::error`.
+- The error type of `zcash_client_backend::data_api::wallet::decrypt_and_store_transaction`
+  has been changed; the error type produced by the provided `WalletWrite` instance is
+  returned directly.
+
+### Deprecated
+- `zcash_client_backend::data_api::wallet::create_spend_to_address` has been
+  deprecated. Use `zcash_client_backend::data_api::wallet::spend` instead. If
+  you wish to continue using `create_spend_to_address`, note that the arguments
+  to the function has been modified to take a unified spending key instead of a
+  Sapling extended spending key, and now also requires a `min_confirmations`
```

Comment:
```suggestion
  to the function have been modified to take a unified spending key instead of a
  Sapling extended spending key, and it now also requires a `min_confirmations`
```